.github/workflows/cicd.yaml

```diff
@@ -13,6 +13,25 @@ env:
   RELEASE_IMAGE: securesystemsengineering/connaisseur
 
 jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Connaisseur image
+        run: |
+          sed -i "s+TAG =.*+TAG = ${RELEASE_VERSION}+" Makefile
+          sed -i "s+IMAGE_NAME =.*+IMAGE_NAME = ${RELEASE_IMAGE}+" Makefile
+          make docker
+      - name: Save images
+        run: |
+          mkdir images
+          docker save ${RELEASE_IMAGE}:${RELEASE_VERSION} -o images/${GITHUB_SHA}_image.tar
+          docker save ${RELEASE_IMAGE}:helm-hook -o images/${GITHUB_SHA}_hook.tar
```

Comment:
Maybe out of scope for this PR, but we are not versioning the helm-hook here, so we technically can't change the hook script without changing naming here (or we break older versions on redeployment), right?

Reply:
added to release discussion